api/v1/verticaautoscaler_types.go

```diff
@@ -281,9 +281,16 @@ type PrometheusSpec struct {
 
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=false
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch","urn:alm:descriptor:com.tectonic.ui:advanced"}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
```

Comment:
Why was it removed?

Reply:
Because there is no "advanced" section yet for VerticaAutoscaler in Openshift. VerticaAutoscaler is not big enough to have an advanced section.